schema_source/definitions.json

```diff
@@ -56,6 +55,25 @@
         "rtt": {
           "type": "number",
           "description": "The last known latency between the NATS Server and the Client in nanoseconds"
+        },
+        "jwt": {
```

Comment:
@matthiashanel can you please review the descriptions here and clarify? These key names are not super descriptive :)